portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -563,90 +570,104 @@ public void addGroupUsers(long groupId, List<User> users)
 			userIds.add(user.getUserId());
 		}
 
-		addGroupUsers(groupId, ArrayUtil.toLongArray(userIds));
+		return addGroupUsers(groupId, ArrayUtil.toLongArray(userIds));
 	}
 
 	/**
 	 * Adds the users to the group.
 	 *
 	 * @param groupId the primary key of the group
 	 * @param userIds the primary keys of the users
+	 * @return <code>true</code> if at least an association between the ${groupId} and the ${userIds} is added; <code>false</code> if all were already added
 	 */
 	@Override
-	public void addGroupUsers(long groupId, long[] userIds)
+	public boolean addGroupUsers(long groupId, long[] userIds)
 		throws PortalException {
 
-		_groupPersistence.addUsers(groupId, userIds);
+		if (!super.addGroupUsers(groupId, userIds)) {
+			return false;
+		}
```

Comment:
@marianoalvarosaiz, for the kind of source pattern, due to *BaseImpl invoke *Persistence, before the pr, if it used *Persistence, please also use ***Persistence**, instead of using **super**.

Reply:
@yuhai I'm not so sure about this one. According to [Tina's second point](https://github.com/liferay-core-infra/liferay-portal/pull/5027#issuecomment-1646095631) I believe I should be even more strict and change every occurrence (related to my code only) of ***Persistence** to **super**. Don't you think so?

Reply:
@marianoalvarosaiz, I checked [Tina's second point](https://github.com/liferay-core-infra/liferay-portal/commit/f955c5eeab57b6ca82e0d9d0567573aa9ef6f001), in this commit, the original usage used **super**, and you changed it for ***Persistence**. Actually, I think my meaning is the same as Tina. In the current commit, the original usage used _groupPersistence(***Persistence**), you used **super**. We shouldn't change the original usage. If the original usage is **Persistence**, please used **Persistence**. If the original usage is **super**, please used **super**